docs/advanced.md

```diff
@@ -362,3 +362,41 @@ is set to `None` or it cannot be inferred from it, HTTPX will default to
   ...
 }
 ```
+
+## SSL certificates
+
+!!! warning
+    This is an advanced section. You most likely won't need to use the features documented here, unless you know what you're doing.
```

Comment:
Nit: I'd drop the "unless you know what you're doing" given you've said "most likely won't need".

Reply:
Well, only keeping "You most likely won't need to use the features documented here" feels a bit like "we document these features, but they aren't useful!". 😕 

I guess the core problem here is that "unless you know what you're doing" might be a bit patronizing?

Anyway, would you be OK with…?

> This is an advanced section. You typically won't need this, unless you have special needs regarding your SSL configuration.